test/net.test.js

```diff
@@ -143,828 +144,87 @@ describe("Calculate net salary", function () {
-                        { name: 'Tax level 2', rate: 0.1, amount: 500000 },
-                        { name: 'Tax level 3', rate: 0.15, amount: 1200000 },
-                        { name: 'Tax level 4', rate: 0.2, amount: 2800000 },
-                        { name: 'Tax level 5', rate: 0.25, amount: 5000000 },
-                        { name: 'Tax level 6', rate: 0.3, amount: 8379000 }
-                    ],
-                    totalTax: 18129000,
-                    netSalary: 72801000
-                };
-                assert.deepStrictEqual(actualPayslip, expectedPayslip);
-            })
-        })
-    });
-
-    describe('Check Net salary with 7 rates of taxes: ', function () {
-
-        /**
-         * gross = NET + tax + inssurance
-         * Case 1: inssurance + dont have taxableIncome
-         * => NET = Gross - gross*10.5%
-         * Case 2: inssurance + have taxableIncome ()
-         * => NET = Gross - tax - gross*10.5% - dependents
-         *
-         */
-        test('It applies no tax', function () {
-            //afterInsurance below 11 million is tax-exempt.
-            //Gross - 10.5% Gross < 11m (Personal deductions)
-            // Gross < 12.29m
-
-            const actualPayslip = net(12_000_000);
+            // Verify
             const expectedPayslip = {
-                gross: 12_000_000,
+                gross: 93_600_000,
                 dependents: 0,
                 dependentDeductionAmount: 0,
-                region: 1,
-                insurances: [
-                    {
-                        amount: 960_000,
-                        name: 'Social Insurance 8%'
-                    },
-                    {
-                        amount: 180_000,
-                        name: 'Health Insurance 1.5%'
-                    },
-                    {
-                        amount: 120_000,
-                        name: 'Unemployment Insurance 1%'
-                    },
-                ],
-                totalInsurance: 1260000,
-                afterInsurance: 10_740_000,
-                taxes: [],
-                totalTax: 0,
-                netSalary: 10_740_000
+                region: region,
+                insurances: mockedInsurance.insurances,
+                totalInsurance: mockedInsurance.total,
+                afterInsurance: actualPayslip.gross - mockedInsurance.total,
+                taxes: mockedTaxes.rates,
+                totalTax: mockedTaxes.totalTax,
+                netSalary: actualPayslip.afterInsurance - mockedTaxes.totalTax
             };
             assert.deepStrictEqual(actualPayslip, expectedPayslip);
         })
 
-        test('It applies tax rate 1 only', function () {
-            //11m < Gross - 10.5% Gross < 11m + 5m (tax level 1)
-            //12.29m < Gross <17.87
+        test('It should correctly deduct default values for total Insurance and total Tax ', function () {
```

Comment:
Pls make the test case name more **descriptive**. Something meaning that the insurancesCalc call with default args